staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -151,7 +151,12 @@ func ValueOrDefault(intOrPercent *IntOrString, defaultValue IntOrString) *IntOrS
 	return intOrPercent
 }
 
-func GetValueFromIntOrPercent(intOrPercent *IntOrString, total int, roundUp bool) (int, error) {
+//GetScaledValueFromIntOrPercent is meant to replace GetValueFromIntOrPercent.
```

Comment:
Nit: please keep a space between the `//` and the words that follow to match the style of the code base.